src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
+     :active (apply + (map count-map ["Submitted" "Running" "Aborting"]))
+     :terminal (apply + (map count-map ["Failed" "Succeeded" "Aborted"]))}))
+
+(react/defc GlobalSubmissionStatus
+  {:render
+   (fn [{:keys [state]}]
+     (let [status-error (:status-error @state)
+           status-counts (:status-counts @state)]
+       (cond
+         status-error [:span {} [:br {}] "Submissions: " status-error]
+         status-counts [:span {} [:br {}] "Submissions: " (:queued status-counts) " Queued; " (:active status-counts) " Active"]
+         :else [:span {}])))
+   :component-did-mount
+   (fn [{:keys [this locals]}]
+     ;; Call once for initial load
+     (react/call :load-data this)
+     ;; Add a long-polling call for continuous updates
+     (swap! locals assoc :interval-id (js/setInterval #(react/call :load-data this) (config/submission-status-refresh))))
+   :component-will-unmount
+   (fn [{:keys [locals]}]
+     (js/clearInterval (:interval-id @locals)))
+   :load-data
+   (fn [{:keys [state]}]
+     (endpoints/call-ajax-orch
+       {:endpoint (endpoints/submissions-queue-status)
+        :on-done (fn [{:keys [success? status-text get-parsed-response]}]
+                   (if success?
+                     (swap! state assoc :status-counts (queue-status-counts (get-parsed-response)))
```

Comment:
in the success case, I think you also need to clear the previous error from the state. Here's my use case:
1) Load a UI page, get successful queue status information, notice that it reloads every 60 secs, happy
2) Suddenly rawls has a transient error, and one of the reloads gets an exception. We display the exception.
3) As soon as rawls returns, we STILL display the exception - because the exception is still in the state.

I tested this by running a local stack, loading a UI page, and manually changing the rawls queueStatus endpoint behind the scenes to throw an error, then re-edited it to be successful.